build.gradle

```diff
@@ -241,7 +341,7 @@ project('serializers') {
     }
 
     jar {
-        manifest {}
+        zip64=true
```

Comment:
AFAIK empty manifest is default and unused. its equivalent to not specifying manifest at all. 

zip64 was added because compiler was complaining about number of entries in the archive being higher than a threshold whereby requiring zip64.